### ISSUE TITLE
master branch warmup learning rate is not bias lr falls from 0.1 to lr0

### ISSUE THREAD

**Issue by challow0** [ultralytics/yolov5#8352]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### Bug

- when adding to optimizer groups, `g[2]` is not the last to add. But in warmup it is still `j == 2`
- This is the master branch
https://github.com/ultralytics/yolov5/blob/0537e8dd13859c4b44db3bf6f39b9ff20eaf163b/train.py#L153-L171

- This is the v6.1 tag branch
https://github.com/ultralytics/yolov5/blob/3752807c0b8af03d42de478fbcbf338ec4546a6c/train.py#L156-L173

- This is the master branch warmup
https://github.com/ultralytics/yolov5/blob/0537e8dd13859c4b44db3bf6f39b9ff20eaf163b/train.py#L336-L338

### Are you willing to submit a PR?

- [x] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@challow0 thanks for the bug report! It looks like you are correct, 338 should have been updated and has not been.

**Comment by glenn-jocher** [comment]
@challow0 good news 😃! Your original issue may now be fixed ✅ in PR #8356. This sets the bias LR warmup to index 0 which aligns with the bias param group.

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
